.github/workflows/gwos-evm-ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
 
     - name: Run Ethereum Test
       working-directory: gwos-evm/polyjuice-tests
-      run: RUST_LOG=info,gw_generator=debug cargo test --release --test ethereum_test -- ethereum_test --nocapture
```

Comment:
Why skip [`ethereum_test`](https://github.com/godwokenrises/godwoken/blob/31c4f568ebf92412ecc8ca60c2bef36a997022ac/gwos-evm/polyjuice-tests/tests/ethereum_test.rs#L472-L509)?

Reply:
Here is a simple rename. We never run the whole Ethereum test. Just the VMTests directory.

Reply:
I see